service/matching/task_queue_partition_manager.go

```diff
@@ -779,10 +795,16 @@ func (pm *taskQueuePartitionManagerImpl) getPhysicalQueuesForAdd(
 	directive *taskqueuespb.TaskVersionDirective,
 	forwardInfo *taskqueuespb.TaskForwardInfo,
 	runId string,
-) (pinnedQueue physicalTaskQueueManager, syncMatchQueue physicalTaskQueueManager, userDataChanged <-chan struct{}, err error) {
+) (spoolQueue physicalTaskQueueManager, syncMatchQueue physicalTaskQueueManager, userDataChanged <-chan struct{}, err error) {
 	wfBehavior := directive.GetBehavior()
 	deployment := directive.GetDeployment()
 
+	perTypeUserData, perTypeUserDataChanged, err := pm.getPerTypeUserData()
```

Comment:
I would still call it
```suggestion
	perTypeUserData, userDataChanged, err := pm.getPerTypeUserData()
```
the channel broadcast is still when any user data changes, right?

Reply:
yes. Renamed.